int8 meta4
